de-lib/src/main/java/org/iplantc/de/diskResource/client/DiskResourceView.java

```diff
@@ -46,6 +48,10 @@
 
     void setWestPanelWidth(String width);
 
+    ColumnModel<DiskResource> getColumns();
+
+    void setColumnPreferences(Map<String, String> preferences);
+
```

Comment:
Please create Javadoc comments describing the purpose of each new public method.

Reply:
👍